src/router.js

```diff
@@ -30,11 +30,13 @@ export const mountRoutes = (app, provider) => {
       const notifications = messages[req.query.notification]
         ? [messages[req.query.notification]]
         : [];
+      const acr = req.session.infos.acr_values;
```

Comment:
what if infos is not defined ?

![image](https://user-images.githubusercontent.com/4190728/68229469-a0b91580-fff7-11e9-8bf4-12b5e0ffde75.png)

Reply:
yes handled in further commits